### PR TITLE
IDENTITY-2737 Code modifications

### PR DESCRIPTION
**Pull request by chanukaranaba**

### CODE REVIEW

File: components/claim-mgt/org.wso2.carbon.claim.mgt.ui/src/main/resources/web/claim-mgt/claim-view.jsp [hunk 1]
```diff
@@ -260,13 +262,17 @@
                         } else {
                             ClaimAttributeDTO[] attrMap = claims[j].getMappedAttributes();
                             if (attrMap != null) {
-                                String val = "";
+                                StringBuilder stringBuilder = new StringBuilder();
```
Comment (darshanasbg) [inline components/claim-mgt/org.wso2.carbon.claim.mgt.ui/src/main/resources/web/claim-mgt/claim-view.jsp]: use meaningful variable names

**Comment by chanukaranaba** [comment]
Code modifications with adding meaning full variable name.

File: components/claim-mgt/org.wso2.carbon.claim.mgt.ui/src/main/resources/web/claim-mgt/claim-view.jsp [hunk 2]
```diff
@@ -32,6 +32,8 @@
 <%@page import="org.wso2.carbon.user.core.UserCoreConstants" %>
 <%@page import="org.wso2.carbon.utils.ServerConstants" %>
 <%@ page import="java.util.ResourceBundle" %>
+<%@ page import="com.ctc.wstx.util.StringUtil" %>
```
Comment (johannnallathamby) [inline components/claim-mgt/org.wso2.carbon.claim.mgt.ui/src/main/resources/web/claim-mgt/claim-view.jsp]: unused import

**Comment by chanukaranaba** [comment]
Unused import statement removed

**Pull request merged**